SmartDeviceLink/SDLLifecycleManager.m

```diff
@@ -522,11 +523,23 @@ - (void)sdl_sendAppIcon:(nullable SDLFile *)appIcon withCompletion:(void (^)(voi
 
 #pragma mark Sending Requests
 
-- (void)sendRequest:(SDLRPCRequest *)request {
-    [self sendRequest:request withResponseHandler:nil];
+- (void)sendRequest:(__kindof SDLRPCMessage *)request {
+    if ([request isKindOfClass:SDLRPCRequest.class]) {
+        SDLRPCRequest *requestRPC = (SDLRPCRequest *)request;
+        [self sendRequest:requestRPC withResponseHandler:nil];
+    } else if ([request isKindOfClass:SDLRPCResponse.class] || [request isKindOfClass:SDLRPCNotification.class]) {
+        [self sdl_sendRPC:request];
+    } else {
+        NSAssert(false, @"The request should be of type `Request`, `Response` or `Notification");
+    }
+}
+
+- (void)sdl_sendRPC:(__kindof SDLRPCMessage *)rpc {
+    SDLRPCResponseOperation *op = [[SDLRPCResponseOperation alloc] initWithConnectionManager:self rpc:rpc];
```

Comment:
Renamed `SDLRPCResponseOperation` to `SDLAsynchronousRPCOperation`

I created a separate class, `SDLRPCResponseOperation`, because `SDLAsynchronousRPCRequestOperation` and `SDLAsynchronousRPCRequestOperation` both  expect a response back from Core in order to "finish" an operation. I tried modifying the `SDLAsynchronousRPCRequestOperation` to handle all three but I had to add a bunch on non-intuitive handling to `SDLLifecycleManager` and `SDLAsynchronousRPCRequestOperation` so I thought it would be better to create a class for responses and notifications which do not get responses back from Core.